oktapam/client/team_settings.go

```diff
@@ -0,0 +1,117 @@
+package client
+
+import (
+	"context"
+	"fmt"
+	"github.com/okta/terraform-provider-oktapam/oktapam/constants/attributes"
+	"github.com/okta/terraform-provider-oktapam/oktapam/logging"
+	"github.com/okta/terraform-provider-oktapam/oktapam/utils"
+	"net/http"
+	"net/url"
+)
+
+type TeamSettings struct {
+	Team                            *string `json:"team,omitempty"`
+	ReactivateUsersViaIDP           *bool   `json:"reactivate_users_via_idp,omitempty"`
+	ApproveDeviceWithoutInteraction *bool   `json:"approve_device_without_interaction,omitempty"`
+	PostDeviceEnrollmentURL         *string `json:"post_device_enrollment_url,omitempty"`
+	PostLogoutURL                   *string `json:"post_logout_url,omitempty"`
+	PostLoginURL                    *string `json:"post_login_url,omitempty"`
+	UserProvisioningExactUserName   *bool   `json:"user_provisioning_exact_username,omitempty"`
+	ClientSessionDuration           *int    `json:"client_session_duration,omitempty"`
+	WebSessionDuration              *int    `json:"web_session_duration,omitempty"`
+	IncludeUserSID                  *string `json:"include_user_SID,omitempty"`
+}
+
+func (s TeamSettings) ToResourceMap() map[string]interface{} {
```

Comment:
Sajan upgraded Go version to 1.18 (https://github.com/okta/terraform-provider-oktapam/pull/96). With that change, you can replace all interface{} occurrences in the code to any. 

Update your code from master. Then, to change all occurrences you can also do - gofmt -r 'interface{} -> any' -w *.go

Reply:
I think you didn't replace interface{} to any.